src/Application/Interfaces/IVacationService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Domain.Entities;
+
+namespace Application.Interfaces
+{
+    public interface IVacationService : IDisposable
+    {
+        Task<IEnumerable<Vacation>> GetAll();
```

Comment:
Add Async at the end of all async methods -> GetAllAsync()

Reply:
@vasiliv & @Dmitry925  advice me to make it not async